src/udf/src/external.rs

```diff
@@ -42,7 +44,10 @@ impl ArrowFlightUdfClient {
 
     /// Connect to a UDF service lazily (i.e. only when the first request is sent).
     pub fn connect_lazy(addr: &str) -> Result<Self> {
-        let conn = tonic::transport::Endpoint::new(addr.to_string())?.connect_lazy();
+        let conn = tonic::transport::Endpoint::new(addr.to_string())?
```

Comment:
Should we also update `ArrowFlightUdfClient::connect`? (I'm not sure if it's possible to reuse the code)